cmd/utils/flags.go

```diff
@@ -1363,6 +1389,13 @@ func setDataDir(ctx *cli.Context, cfg *node.Config) {
 	}
 }
 
+func setRaftLogDir(ctx *cli.Context, cfg *node.Config) {
+	switch {
+	case ctx.GlobalIsSet(RaftLogDirFlag.Name):
+		cfg.RaftLogDir = ctx.GlobalString(RaftLogDirFlag.Name)
+	}
+}
+
```

Comment:
If the raft directory isn't set, then it could fallback to the data directory (which it can pull from the node config).

Reply:
a new check added, if the directory not set, it fetches it from datadir option.